src/components/prompt_box.tsx

```diff
@@ -0,0 +1,53 @@
+import { title } from 'process';
+import { SetStateAction, useState } from 'react';
+import { Button } from 'react-bootstrap';
+import Modal from 'react-bootstrap/Modal';
+
+interface InputModalProps {
```

Comment:
Since this only allows a string input, maybe StringINputModal would be a more descriptive name for this component.  The file name should match the component name.